lmdeploy/pytorch/models/internlm2.py

```diff
@@ -171,6 +171,176 @@ def forward(
         )
 
 
+class PatchedInternLM2AttentionAscend(nn.Module):
+
+    def _load_weights(self, loader, rank: int, world_size: int,
```

Comment:
`_laod_weights` is the same as `PatchedInternLM2Attention::_laod_weights`.
Can `PatchedInternLM2AttentionAscend` reuse it?

Reply:
The situation is the same as in `LlamaAttentionAscend` and `LlamaAttention`.